.github/workflows/ci.yml

```diff
@@ -60,6 +57,13 @@ jobs:
           CONNECTED_APP_SECRET_ID: ${{ secrets.E2E_TEST_CONNECTED_APP_SECRET_ID }}
           CONNECTED_APP_SECRET_VALUE: ${{ secrets.E2E_TEST_CONNECTED_APP_SECRET_VALUE }}
 
+      - name: Build Claude MCP Bundle
+        run: |
+          npm run build:manifest:script
+          npm run build:manifest
+          npm ci --omit=dev
```

Comment:
It hadn't occurred to me that all of the dev dependencies were just bloating the MCPB file, by like 10x. This will delete node_modules and won't install the dev dependencies.